docs/gettingstarted/structuring.md

```diff
@@ -30,7 +30,8 @@ All components of @Mod() are written inside the parantheses and chained together
 Components:
 
 `modid` [Required]
-This identifies your mod uniquely to Forge and should probably be the first thing you enter. It's only visible internally.
+This identifies your mod uniquely to Forge and should probably be the first thing you enter. It's only used internally, but can be seen by users who dig around. Other mods also require this ID if they want to find each other. (APIs for example.)
```

Comment:
Nitpicky, but still: periods go after the parentheses, not before: 
"each other (APIs for example)."